packages/react-router-dom/package.json

```diff
@@ -34,7 +34,7 @@
     "prepublish": "node ./tools/build.js",
     "clean": "git clean -fdX .",
     "lint": "eslint modules",
-    "test": "karma start --single-run"
+    "test": "jest"
```

Comment:
Wait, didn't we want to leave Karma for react-router-dom? I think it's what broke --streaming, IIRC. So, that might be an issue if you add it back in...